### DISCUSSION TITLE
posix standart

### DISCUSSION THREAD

**Discussion by seldish-og** [ParanoidUser/yolo#148] [Galaxy Trivia] [answered]
What is posix standart in C?

**Comment by thuongtruong1009** [comment, marked as answer]
In C programming, POSIX refers to the set of standard libraries defined by the POSIX standard that provide system interfaces for C programs. These libraries are used to write portable programs that can be run on any POSIX-compliant operating system.

Some of the key POSIX libraries in C include:

* ***`<unistd.h>`***: Provides access to POSIX system calls related to file I/O, process management, and system configuration.

* ***`<pthread.h>`***: Provides POSIX threads (also known as pthreads) for multi-threaded programming.

* ***`<signal.h>`***: Provides POSIX signals for handling asynchronous events and interrupt handling.

* ***`<time.h>`***: Provides functions for working with dates and times.

* ***`<fcntl.h>`***: Provides POSIX file control options for working with file descriptors.

* ***`<sys/socket.h>`***: Provides POSIX sockets for network programming.

The POSIX standard libraries provide a consistent interface for C programs to access system services, making it easier to write portable code that can be run on different operating systems.
